### ISSUE TITLE
Проблема с прошивкой

### ISSUE THREAD

**Issue by Forldol** [tretyakovsa/Sonoff_WiFi_switch#156]
Здравствуйте, возникла такая проблема. Прошиваю ESP-07 последней прошивкой, всё прошивается нормально, появляется точка доступа, к ней подключается но после внесения любых изменение и перезагрузки устройства получаю вот такое

![2019-04-17 18 48 29](https://user-images.githubusercontent.com/46322267/56302379-497a5300-6142-11e9-956a-e404fe66d39f.PNG)

Что можете посоветовать?

**Comment by renat2985** [comment]
В 90% у вас на ESP установлен Flash Chip который плохо работает с файловой системой. Зайдите на страницу http://192.168.*.*/page.htm?debug 
И покажите Flash Chip

**Comment by Forldol** [comment]
![2019-04-17 21 31 19](https://user-images.githubusercontent.com/46322267/56312411-c0bae180-6158-11e9-853d-de1e1de9d24f.PNG)

**Comment by renat2985** [comment]
Да, видите у вас красным написано Bad Chip. Вы не сможете нормально работать с файловой системой. Как только ESP8266 что-то попытается записать в файловую систему, сразу же сломает этот файл. Об это много писали у нас и много пишут в интернете. Вроде как новое ядро уже научили работать с этими Flash чипами, мы еще не проверяли.
Решения проблемы: Некоторые перепаивают Flash чипы. Или же ждите, возможно со временем все будет работать.

**Comment by Forldol** [comment]
Это печально, подскажите какие чипы памяти туда можно запаять

**Comment by tretyakovsa** [comment]
@Forldol Не торопитесь перепаивать. Нужно проверить будут ли работать ваши модули с новой версией. Скоро выложу новую бету.

**Comment by Forldol** [comment]
@tretyakovsa Напишите пожалуйста сюда когда можно пробовать, я обязательно попробую =)

**Comment by AntKul** [comment]
У меня такая же проблема с модулями ESP-01 из Китая. Ждемс новую версию.

**Comment by Forldol** [comment]
Не дождался, перевел все свои модули на wifi-iot
